dipy/reconst/ivim.py

```diff
@@ -120,37 +125,42 @@ def f_D_star_error(params, gtab, signal, S0, D):
     return signal - f_D_star_prediction([f, D_star], gtab, S0, D)
 
 
-def ivim_model_selector(gtab, fit_method='LM', **kwargs):
+def ivim_model_selector(gtab, fit_method='trf', **kwargs):
     """
-    Selector function to switch between the 2-stage Levenberg-Marquardt based
-    NLLS fitting method (also containing the linear fit): `LM` and the Variable
-    Projections based fitting method: `VarPro`.
+    Selector function to switch between the 2-stage Trust-Region Reflective
+    based NLLS fitting method (also containing the linear fit): `trf` and the
+    Variable Projections based fitting method: `varpro`.
 
     Parameters
     ----------
     fit_method : string, optional
-        The value fit_method can either be 'LM' or 'VarPro'.
-        default : LM
+        The value fit_method can either be 'trf' or 'varpro'.
+        default : trf
 
     """
+    boundsWarning = 'Bounds for this fit have been set from experiments '
+    boundsWarning += 'and literature survey. To change the bounds, please '
+    boundsWarning += 'input your bounds in model definition...'
 
-    if fit_method.lower() == 'lm':
-        return IvimModelLM(gtab, **kwargs)
+    if fit_method.lower() == 'trf':
+        warnings.warn(boundsWarning, UserWarning)
```

Comment:
IIUC, this warning would be raised even if the user set the bounds themselves.

Reply:
Yes!

Reply:
So this warning will always popup whatever the user does? What is the goal of this warning?

Reply:
As discussed in #1817 , it is to warn the users that there is no fixed recipe for the bounds and we provide the users a way to change them.. makes sense?

Reply:
ok, but when the user changes the bounds, the warning will still appear, right? Do we really want that or should we adapt? if the user changes it, no warning otherwise, a warning appears

Reply:
@skoudoro You are right! I have tried to address this in my latest commit :)